target/linux/ipq807x/files/arch/arm64/boot/dts/qcom/ipq8074-sxr80.dts

```diff
@@ -0,0 +1,643 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+/dts-v1/;
+
+#include "ipq8074.dtsi"
+#include "ipq8074-ac-cpu.dtsi"
```

Comment:
If this is IPQ8074 model then this include is incorrect, you need the HK CPU

Reply:
Fixed.